src/main/resources/bidder-config/loyal.yaml

```diff
@@ -0,0 +1,27 @@
+adapters:
+  loyal:
+    enabled: false
+    endpoint: "https://us-east-1.loyal.app/pserver?p={{PlacementId}}&e={{EndpointId}}"
+    pbs-enforces-gdpr: false
+    pbs-enforces-ccpa: false
+    modifying-vast-xml-allowed: false
+    geo-target:
+      - USA
+    meta-info:
+      maintainer-email: "hello@loyal.app"
+      app-media-types:
+        - banner
+        - video
+        - native
+      site-media-types:
+        - banner
+        - video
+        - native
+      supported-vendors: []
+      vendor-id: 0
+    usersync:
+      url: ""
+      redirect-url: ""
+      cookie-family-name: loyal
+      type: redirect
+      support-cors: false
```

Comment:
I suggest removing it until not configured, anyway layout of properties is incorrect here (please check other bidders)